src/scripts/main.js

```diff
@@ -1,3 +1,231 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+
+body.insertAdjacentHTML('beforeend', `
+  <form 
+  selected="#"
+  method="get"
+  class="new-employee-form"
+  >
```

Comment:
```suggestion
  <form 
    selected="#"
    method="get"
    class="new-employee-form"
  >
```

Reply:
what does selected attribute do?

Reply:
This is my typo)  It should be "action=#"